protocol/protocol.proto

```diff
@@ -85,6 +85,9 @@ message StreamEvent {
     /** DelegateExpiry is the time when the delegate signature expires. */
     int64 delegate_expiry_epoch_ms = 6;
 
+    /** Tags are optional, used for external systems to deliver notifications. */
+    Tags tags = 7;
```

Comment:
If these are optional should we mark them as optional in the protobuf file?

Reply:
no, in proto3 all fields are optional by default and `optional` keyword is reused for other functionality